src/Airmap/QJsonWebToken/src/qjsonwebtoken.cpp

```diff
@@ -186,9 +186,9 @@ bool QJsonWebToken::setAlgorithmStr(QString strAlgorithm)
 	// set algorithm
 	m_strAlgorithm = strAlgorithm;
 	// modify header
-	m_jdocHeader = QJsonDocument::fromJson(QObject::trUtf8("{\"typ\": \"JWT\", \"alg\" : \"").toUtf8()
+    m_jdocHeader = QJsonDocument::fromJson(QStringLiteral("{\"typ\": \"JWT\", \"alg\" : \"").toUtf8()
```

Comment:
This doesn't seem to need to use `tr`? Correct?

Reply:
what's the motivation to change it?

Reply:
I've never touched this code...AirMap didn't author it

Reply:
Oh, that's right. This is Gus' code. The calls in this pull are deprecated in Qt 5.15 and won't compile without warnings. These changes get rid of the warnings. This all should be fine then.